notifications.yaml

```diff
@@ -0,0 +1,220 @@
+swagger: "2.0"
+info: 
+  version: "0.0.1"
+  title: "Notification API"
+  description: "Notification API."
+host: "localhost"
+basePath: "/api/v1"
```

Comment:
To put the "/api" prefix is a suggestion that comes from Zalando guidelines: https://zalando.github.io/restful-api-guidelines/naming/Naming.html

This may be useful in case you have private API aside public ones.